### ISSUE TITLE
[Xfce] Gahshomar systray disappears 

### ISSUE THREAD

**Issue by MehdiAria** [Gahshomar/gahshomar#41]
Even after adding gahshomar manually to the Xfce panel, after restarting the machine gahshomar icon disappears from systray and you will need to start it manually each time or add it to startup items while this will not solve the problem in the end.

**Comment by 183amir** [comment]
Gahshomar needs to be running in the background for the icon to work. Why do you say adding Gahshomar to startup apps does not work?

**Comment by MehdiAria** [comment]
It is running in the background but this won't keep running after a restart.
Because even if you add it to startup items to solve the problem mentioned above, this will open the gahshomar window each time you start your machine, and closing the gahshomar window after each restart is kinda annoying.

**Comment by 183amir** [comment]
Yes, I agree that it is annoying to see a window popup at startup. I have the same issue. However, to fix it, I have to refactor the app to a service and a window. The service would be always running in the background. Unfortunately, this will take a lot of time to implement and test.
